OasisDEX/oasis-borrow
# Update vault header

# [Update vault header](https://app.shortcut.com/oazo-apps/story/3855/redesign-update-vault-header)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

Headers are updated in:

- open vault,
- manage vault,
- manage vault in new layout.
  
## How to test 🧪
  <Please explain how to test your changes>

All new headers are closed behind `AutomationBasicBuyAndSell` feature flag. Additionally, if you want to see new manage vault layout, you also need to set up a flag `Automation`.
